Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+pipeline {
+    agent label:"docker", docker: "maven:3.3.9-jdk-7"
+
+    environment {
+      MAVEN_OPTS = "-XX:MaxPermSize=256m -Xmx1024m"
+    }
+
+    stages {
+        stage("Build") {
+            sh 'mvn clean install -Dmaven.test.failure.ignore=true'
```

Comment:
Yes, `=true` can be omited but, it [disturbs my circles](https://en.wikipedia.org/wiki/Noli_turbare_circulos_meos!) ;)

Added `-B`